.eslintrc.js

```diff
@@ -25,4 +25,7 @@ module.exports = {
             },
         },
     },
+    globals: {
+        __DEV__: 'readonly',
+    },
```

Comment:
What is this for?

Reply:
`__DEV__` is a global variable in RN, and this tells ESLint that so that we can remove some unnecessary ESLint suppressions. Not too closely related to this PR, just a bit of housekeeping.